### DISCUSSION TITLE
documentation for bolt 4 contact form extension is wrong. How can I configure my contact form?

### DISCUSSION THREAD

**Discussion by tskraft** [bolt/core#2658] [Q&A]
Hello,

I use bolt 4 and want to setup a form on my website. 

I installed the forms extension for bolt 4 (https://github.com/bolt/forms). In the docs folder of the unzipped extension I found a file named creating_new_forms.md

Cite from that file:

All the configuration required to customise the contact form, or build
your very own Bolt form is located in `config/extensions/bolt-boltforms.yaml`.

But I didn't find this file after installation in the promised folder. In the downloaded zip I found a file config.yaml in the config folder. This looks like the needed file.

So how does this extension work? How can I create  a contact form in my twig template? Alternative: I managed to setup a form in a pure symfony app. How can I get that over to boltcms 4?

Thank you very much for your help!

**Comment by I-Valchev** [comment]
Hi @tskraft , to install the extension, you need to run `composer require bolt/forms` from the root folder of your Bolt website. 

here is the full documentation: https://bolt.github.io/forms/

Can you try that?

**Reply by Fabiomad85** [reply]
the documentation is really poor and basically useless for the basic cases, like a hidden field
